packages/camelot/src/abi.ts

```diff
@@ -0,0 +1,247 @@
+    ],
+    name: 'removeLiquidityETHSupportingFeeOnTransferTokens',
+    outputs: [{ internalType: 'uint256', name: 'amountETH', type: 'uint256' }],
+    stateMutability: 'nonpayable',
+    type: 'function',
+  },
+  {
+    inputs: [
+      { internalType: 'address', name: 'token', type: 'address' },
+      { internalType: 'uint256', name: 'liquidity', type: 'uint256' },
+      { internalType: 'uint256', name: 'amountTokenMin', type: 'uint256' },
+      { internalType: 'uint256', name: 'amountETHMin', type: 'uint256' },
+      { internalType: 'address', name: 'to', type: 'address' },
+      { internalType: 'uint256', name: 'deadline', type: 'uint256' },
+      { internalType: 'bool', name: 'approveMax', type: 'bool' },
+      { internalType: 'uint8', name: 'v', type: 'uint8' },
+      { internalType: 'bytes32', name: 'r', type: 'bytes32' },
+      { internalType: 'bytes32', name: 's', type: 'bytes32' },
+    ],
+    name: 'removeLiquidityETHWithPermit',
+    outputs: [
+      { internalType: 'uint256', name: 'amountToken', type: 'uint256' },
+      { internalType: 'uint256', name: 'amountETH', type: 'uint256' },
+    ],
+    stateMutability: 'nonpayable',
+    type: 'function',
+  },
+  {
+    inputs: [
+      { internalType: 'address', name: 'token', type: 'address' },
+      { internalType: 'uint256', name: 'liquidity', type: 'uint256' },
+      { internalType: 'uint256', name: 'amountTokenMin', type: 'uint256' },
+      { internalType: 'uint256', name: 'amountETHMin', type: 'uint256' },
+      { internalType: 'address', name: 'to', type: 'address' },
+      { internalType: 'uint256', name: 'deadline', type: 'uint256' },
+      { internalType: 'bool', name: 'approveMax', type: 'bool' },
+      { internalType: 'uint8', name: 'v', type: 'uint8' },
+      { internalType: 'bytes32', name: 'r', type: 'bytes32' },
+      { internalType: 'bytes32', name: 's', type: 'bytes32' },
+    ],
+    name: 'removeLiquidityETHWithPermitSupportingFeeOnTransferTokens',
+    outputs: [{ internalType: 'uint256', name: 'amountETH', type: 'uint256' }],
+    stateMutability: 'nonpayable',
+    type: 'function',
+  },
+  {
+    inputs: [
+      { internalType: 'address', name: 'tokenA', type: 'address' },
+      { internalType: 'address', name: 'tokenB', type: 'address' },
+      { internalType: 'uint256', name: 'liquidity', type: 'uint256' },
+      { internalType: 'uint256', name: 'amountAMin', type: 'uint256' },
+      { internalType: 'uint256', name: 'amountBMin', type: 'uint256' },
+      { internalType: 'address', name: 'to', type: 'address' },
+      { internalType: 'uint256', name: 'deadline', type: 'uint256' },
+      { internalType: 'bool', name: 'approveMax', type: 'bool' },
+      { internalType: 'uint8', name: 'v', type: 'uint8' },
+      { internalType: 'bytes32', name: 'r', type: 'bytes32' },
+      { internalType: 'bytes32', name: 's', type: 'bytes32' },
+    ],
+    name: 'removeLiquidityWithPermit',
```

Comment:
Dumb q: If we include the other functions inside the abi, is it possible that they could trigger a successful action? Like if the params match

e.g. Can this `removeLiquidity` fx trigger a completion for a `swap` action if it had matching params?

Reply:
Yeah that's possible - I just narrowed the included ABIs that's easier than seeing if what you're describing could happen lol